mod_reforged/hooks/config/strings.nut

```diff
@@ -2077,8 +2086,8 @@ foreach (vanillaDesc in vanillaDescriptions)
 				Type = ::UPD.EffectType.Passive,
 				Description = [
 					"Gain the [Decapitate|Skill+decapitate] skill with swords.",
-					"Gain all the perks of the Cleaver perk group when using a sword. Swords now additionally qualify as Cleavers for you for the purposes of these perks.",
-					"Attacks from swords inflict [Bleeding|Skill+bleeding_effect] on the target."
+					"Gain all the perks of the Cleaver perk group when using a sword. Swords now additionally qualify as Cleavers for the purposes of these perks.",
+					"Attacks from swords inflict [Bleeding|Skill+bleeding_effect] as if they were Cleavers."
```

Comment:
No need for this addition imo.

Reply:
My thought was this added some clarity as to how many stacks of bleed would be added per hit. What do you think?

Reply:
I think that part is implied quite a bit by the entire perk description's context so it seem superfluous.